src/sentry/tasks/relay.py

```diff
@@ -47,12 +48,17 @@ def update_config_cache(generate, organization_id=None, project_id=None, update_
         projects = Project.objects.filter(organization_id=organization_id)
```

Comment:
Can we run a `get_many_from_cache` here?

Reply:
A get_many_from_cache is only useful here if you have multiple project IDs and want to fetch the projects for those.

get_from_cache and get_many_from_cache essentially can't be used for anything other than primary keys and other unique identifiers.

Reply:
👍 you're even writing about this in the PR.